src/views/Player.vue

```diff
@@ -352,6 +315,13 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.red {
```

Comment:
Where is this 'red' class used?

Reply:
https://github.com/w3champions/website/blob/e3cf691a3b2a9dd2f3e58ea6b1c06044995dd13c/src/views/Player.vue#L50

I could only find it used once in `Player.vue` so I moved it from global to scoped for the component. It was messing up the red `L` icons.